### ISSUE TITLE
Help wanted Krita gmic-qt

### ISSUE THREAD

**Issue by bloodywing** [bloodywing/bloody#2]
I'm still kind of a newbie with ebuild and seek help to make an ebuild for gmic qt.
There is a current one for gmic and gimp https://packages.gentoo.org/packages/media-gfx/gmic but this one is without the qt part. Building gmic is kind of a mess and not a simple configure, make process.

Help for this is very welcome.

https://github.com/c-koi/gmic-qt

**Comment by Lucretia** [comment]
Gentoo's gmic package already includes it.

```
$ equery f media-gfx/gmic
 * Searching for gmic in media-gfx ...
 * Contents of media-gfx/gmic-2.7.2:
/usr
/usr/bin
/usr/bin/gmic
/usr/bin/gmic_krita_qt
/usr/bin/gmic_qt
/usr/include
...
```

**Comment by bloodywing** [comment]
oh thanks :). I opened this when gmic hadn't have the ebuild for it. Looks like it got fixed with gmic 2.4.3 here https://bugs.gentoo.org/629194
